src/Microsoft.AspNet.FileProviders.Physical/project.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.0.0-*",
+    "description": "Implementation of ASP.NET 5 physical file provider abstractions.",
+    "dependencies": {
```

Comment:
Right I see that. Otherwise ones consuming these packages will have to include the file system interfaces in their projects.